docker-environment-driver/src/main/resources/isolatedDockerConfiguration/Dockerfile

```diff
@@ -0,0 +1,80 @@
+FROM fedora
+MAINTAINER Marek Novotny <mnovotny@redhat.com>
+
+# install java and zip
+RUN yum -y update && yum clean all
+RUN yum -y install tar gzip java java-devel && yum clean all
```

Comment:
we could, even it is much better to use default installed java from base image - in this case it is java 8. We can fairly easily replace this later if for instance we need to use Oracle jdk or other vendor installations.